rust/src/controllers/emergency.rs

```diff
@@ -128,15 +128,15 @@ fn build_emergency_notifications(
     sender_details: &UserDetails,
     rec_username: &String,
     state: &UserState,
-) -> (Vec<PushNotification>, Vec<Email>) {
+) -> (Vec<PushNotification>, Option<Email>) {
     get_user_details(rec_username, conn)
         .ok()
         .flatten()
-        .map_or((vec![], vec![]), |rec_details| {
+        .map_or((vec![], None), |rec_details| {
             let notifications =
                 build_emergency_notification(conn, &sender_details, &rec_details, &state);
```

Comment:
Is this building SMS and push notifications?

Reply:
Its only building push notifications and emails for now, its not using the phone number to send SMS

Reply:
This is how it was in the node implementation when I made the translation